samples/Mvc.RenderViewToString/Views/EmailPartial.cshtml

```diff
@@ -0,0 +1,4 @@
+﻿@inject Mvc.RenderViewToString.EmailReportHelper ReportHelper
+
+    This is the partial view content.<br />
+    @ReportHelper.GenerateReport(Model.UserData1, Model.UserData2).<br />
```

Comment:
Yeah perhaps no urgency on adding much more to this sample just yet. The goal right now is to get something functional out the door, let the masses know (it's one of the most popular requests for MVC/Razor), and see what people think.